src/main/java/graphql/introspection/Introspection.java

```diff
@@ -398,7 +398,7 @@ private static String printDefaultValue(InputValueWithState inputValueWithState,
                     .name("ofType")
                     .type(typeRef("__Type")))
             .field(newFieldDefinition()
-                    .name("specifiedByUrl")
+                    .name("specifiedByURL")
```

Comment:
This is wrong in both cases

https://spec.graphql.org/draft/#sec--specifiedBy

```
directive @specifiedBy(url: String!) on SCALAR
```

It probably should be `specifiedBy` as a field name

Reply:
Even if we go with this new name  - it a breaking change - anyone using it today would be broken

We need to add a new field and deprecate the old if we changed.

What does graphql-js do here?

Reply:
> This is wrong in both cases
> 
> https://spec.graphql.org/draft/#sec--specifiedBy
> 
> ```
> directive @specifiedBy(url: String!) on SCALAR
> ```
> 
> It probably should be `specifiedBy` as a field name

unless I am mistaken, the field is the one here: https://spec.graphql.org/draft/#sec-Schema-Introspection.Schema-Introspection-Schema

> Even if we go with this new name - it a breaking change - anyone using it today would be broken
> 
> We need to add a new field and deprecate the old if we changed.
> 
> What does graphql-js do here?

they currently only support the `specifiedByURL` field: https://github.com/graphql/graphql-js/blob/main/src/type/introspection.ts#L256 but expressed the same concern here: https://github.com/graphql/graphql-js/issues/3156 . I see no issue with keeping `specifiedByUrl` and possibly deprecating it, similar to how you extended default introspection behavior with IntrospectionWithDirectivesSupport. But I see no similar approach in graphql-js.

Reply:
Let's put both fields in, deprecate the old and laugh about it on 20 years